packages/framework/fluid-framework/README.md

```diff
@@ -13,34 +34,45 @@ When taking a dependency on a Fluid Framework library, we recommend using a `^`
 While Fluid Framework libraries may use different ranges with interdependencies between other Fluid Framework libraries,
 library consumers should always prefer `^`.
 
-<!-- prettier-ignore-end -->
+## Installation
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+To get started, install the package by running the following command:
 
-## Contents
+```bash
+npm i fluid-framework
+```
 
-The `fluid-framework` package consists primarily of two portions: the `IFluidContainer` and a selection of distributed data structures (DDSes).
+## API Documentation
 
-### IFluidContainer
+API documentation for **fluid-framework** is available at <https://fluidframework.com/docs/apis/fluid-framework>.
```

Comment:
Link is not working (yet?). Does it need to be the api/v2 URL, maybe?

Reply:
Ouch, yeah, these will all be wrong until we get redirects working. Let me see if I can add some hard-coded redirects for our SDK packages.